static/js/src/appliance.js

```diff
@@ -0,0 +1,25 @@
+if (typeof appliance !== "undefined") {
+  throw TypeError("Namespace 'appliance' not available");
+}
+
+var appliance = {};
+
+appliance.updateURL = function(linkElement, selectElement) {
+  linkElement.href = window.location.pathname + '/' + selectElement.value;
+}
+
+appliance.downloadFormsInit = function(downloadForms) {
+  if (downloadForms) {
+    downloadForms.forEach(function(container) {
+      const linkElement = container.querySelector('.js-download-link');
+      const selectElement = container.querySelector('.js-platform-select');
+      if (linkElement && selectElement) {
+        selectElement.addEventListener('change', function() {
+          appliance.updateURL(linkElement, selectElement);
```

Comment:
Why not post the option as a form to the thank you page and conditionally download and select tab based on the option?

Reply:
The download pages are going to be at different URLs. 

I was going to see how we did with this review and then just add three links as a no-JS fallback.

Thinking about it, a backend form handler doing the redirect might be a better solution.

Reply:
From the copy docs I've looked at that all follow the same structure. The project page and a thank you page. Is there a different one?